gcloud/language/sentiment.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Definition for Google Cloud Natural Language API sentiment.
+
+Sentiment is the response to an ``analyzeSentiment`` request.
+"""
+
+
+class Sentiment(object):
+    """A Google Cloud Natural Language API sentiment object.
+
+    .. _Sentiment message: https://cloud.google.com/natural-language/\
+                           reference/rest/v1beta1/Sentiment
+
+    See `Sentiment message`_.
+
+    :type polarity: float
+    :param polarity: Polarity of the sentiment in the ``[-1.0, 1.0]`` range.
+                     Larger numbers represent more positive sentiments.
```

Comment:
Hmm, I guess we could point to the [concept docs for sentiment](https://cloud.google.com/natural-language/docs/basics#sentiment-analysis-values):

```
Sentiment 	Sample Values
=========   ===================================
Positive    "polarity": 0.8, "magnitude": 3.0
Negative    "polarity": -0.3, "magnitude": 4.0
Neutral     "polarity": 1.0, "magnitude": 0.0
Mixed 	    "polarity": 0.0, "magnitude": 4.0
```

where "Mixed" means both strong positive and negative statements occur, cancelling one another out.